001-rfcs.md

```diff
@@ -0,0 +1,25 @@
+
+# RFCs
+
+We want to start using RFCs to discuss (and document) architectural decisions.
+
+**Discussion:** https://github.com/DoSomething/rfcs/pull/1
```

Comment:
I really dig the meta-ness of this self-reference, within the meta RFC for the RFC! ♻️